src/contractTest/java/uk/gov/hmcts/reform/professionalapi/provider/ProfessionalExternalUserControllerProviderTest.java

```diff
@@ -193,6 +205,73 @@ private ProfessionalUser setupInteractionsForProfessionalUser() throws JsonProce
+        SecurityContextHolder.setContext(securityContext);
+        when(securityContext.getAuthentication().getPrincipal()).thenReturn(jwt);
+        when(idamRepositoryMock.getUserInfo(anyString())).thenReturn(UserInfo.builder().uid("someUid")
+                .roles(Arrays.asList("pui-case-manager")).build());
+
+        String name = "name";
+        String sraId = "sraId";
+        String companyNumber = "companyNumber";
+        String companyUrl = "companyUrl";
+
+        ProfessionalUser professionalUser = getProfessionalUser(name, sraId, companyNumber, companyUrl);
+        professionalUser.setEmailAddress("someUserIdentifier");
+
+        UserProfile profile = getUserProfile();
+
+        GetUserProfileResponse userProfileResponse = new GetUserProfileResponse(profile, false);
+        String body = objectMapper.writeValueAsString(userProfileResponse);
+
+        when(userProfileFeignClientMock.getUserProfileById("someUserIdentifier"))
+                .thenReturn(Response.builder()
+                        .request(mock(Request.class))
+                        .body(body, Charset.defaultCharset()).status(200).build());
+
+        NewUserResponse newUserResponse = new NewUserResponse();
+        newUserResponse.setUserIdentifier("a123dfgr46");
+        newUserResponse.setIdamStatus("ACTIVE");
+        String newUserResponseBody = objectMapper.writeValueAsString(newUserResponse);
+
+        when(userProfileFeignClientMock.getUserProfileByEmail(anyString())).thenReturn(Response.builder()
+                .request(mock(Request.class)).body(newUserResponseBody, Charset.defaultCharset()).status(200).build());
+
+
+        when(professionalUserRepositoryMock.findByUserIdentifier("someUid")).thenReturn(professionalUser);
+        when(professionalUserRepositoryMock.findByUserIdentifier(PROFESSIONAL_USER_ID)).thenReturn(professionalUser);
+        when(professionalUserRepositoryMock.findByEmailAddress(anyString())).thenReturn(professionalUser);
+
+        when(professionalUserRepositoryMock.findByOrganisation(organisation))
+                .thenReturn(Arrays.asList(professionalUser));
+
+
+        ModifyUserRolesResponse modifyUserRolesResponse = new ModifyUserRolesResponse();
+        RoleAdditionResponse roleAdditionResponse = new RoleAdditionResponse();
+        roleAdditionResponse.setIdamMessage("some message");
+        roleAdditionResponse.setIdamStatusCode("200");
+        modifyUserRolesResponse.setRoleAdditionResponse(roleAdditionResponse);
+
+        String bodyModifyUserRoles = objectMapper.writeValueAsString(modifyUserRolesResponse);
+
+        when(userProfileFeignClientMock.modifyUserRoles(any(), any(), any())).thenReturn(Response.builder()
+                .request(mock(Request.class)).body(bodyModifyUserRoles, Charset.defaultCharset()).status(200).build());
+
+        List<UserConfiguredAccess> allUserConfiguredAccess = new ArrayList<>();
+        allUserConfiguredAccess.add(new UserConfiguredAccess());
```

Comment:
consumer test expecting this new json array field , but i don't see here :
‘userAccessTypes’: [
        {
          ‘jurisdictionId’: ‘jurisdictionId1’,
          ‘organisationProfileId’: ‘organisationProfileId1’,
          ‘accessTypeId’: ‘accessTypeId1’,
          ‘enabled’: true
        }
      ]

Reply:
It is there, but keeps getting overwritten on the broker with each new xui build